ideas42ventures/ideas42ventures.com
# Added the 3 subpages and the Netlify redirect link

Works on #22 

- Added the 3 new subpages underneath the ventures-for-shared-prosperity page so that users can gain more information on what we do.

- Also, added a new Netlify redirect rule so that the `/apply` URL takes users deeper into the Ventures For Shared Prosperity system.

- Fixed the issues where ventures-for-prosperity and focus-area were one and the same.

**How to test:** 

1. The links below should be valid URLs with their respective content on each page

- [ventures-for-shared-prosperity](https://deploy-preview-24--ideas42ventures.netlify.app/ventures-for-shared-prosperity/)

- [focus-area](https://deploy-preview-24--ideas42ventures.netlify.app/ventures-for-shared-prosperity/focus-area)

- [what-we-provide](https://deploy-preview-24--ideas42ventures.netlify.app/ventures-for-shared-prosperity/what-we-provide/)

- [apply](https://deploy-preview-24--ideas42ventures.netlify.app/ventures-for-shared-prosperity/apply)

2. Test the Netlify redirect
[/apply](https://deploy-preview-24--ideas42ventures.netlify.app/ventures-for-shared-prosperity/apply) should redirect to [ventures-for-shared-prosperity/apply/](https://deploy-preview-24--ideas42ventures.netlify.app/ventures-for-shared-prosperity/apply/)
